openassessment/workflow/migrations/0004_reverse_gen_uuid.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models, transaction
+import uuid
+
+
+def gen_uuid(apps, schema_editor):
+    workflow_model_class = apps.get_model('workflow', 'AssessmentWorkflow')
+    total_len = workflow_model_class.objects.count()
+    current_index = 0
+    chunk_size = 1000
+    while current_index < total_len:
+        end_chunk = current + chunk_size if total_len - chunk_size >= current_index else total_len
+        with transaction.atomic():
+            for workflow in workflow_model_class.objects.all()[current_index:end_chunk].iterator():
```

Comment:
Dev note: check whether this change is needed in the recent submissions management command as well